twist_controller does not currently work in simulation

The interface that the twist_controller looks for `tcp/twist...` does not exist in simulation. Add a note in the README usage section to let users know

Error when attempting to switch to twist_controller in sim:
```
[ruby $(which ign) gazebo-7] [ERROR] [1698431966.919867932] [controller_manager]: Can't activate controller 'twist_controller': Command interface with 'tcp/twist.linear.x' does not exist
```

--- a/README.md
+++ b/README.md
@@ -243,6 +243,8 @@ ros2 service call /controller_manager/switch_controller controller_manager_msgs/
 }"
 ```
 
+**Note: the required interface for the `twist_controller` does not currently exist in the gazebo or mock hardware simulation setups. So the `twist_controller` is currently only functional on Kinova hardware.**
+
 Once the `twist_controller` is activated, You can publish Twist messages on the `/twist_controller/commands` topic to command the arm.
 
 For example, you can jog the arm using [Teleop Twist Keyboard](https://index.ros.org/p/teleop_twist_keyboard/github-ros2-teleop_twist_keyboard/) with the following command:

